assembly/src/parsers/ast/u32_ops.rs

```diff
@@ -294,3 +294,18 @@ pub fn parse_u32_rotl(op: &Token, checked: bool) -> Result<Node, AssemblyError>
 
     Ok(node)
 }
+
+pub fn parse_u32assert(op: &Token) -> Result<Node, AssemblyError> {
```

Comment:
minor nit: let's move this function to the top of this file (i.e., before `parse_u32checked_add()`).

Reply:
fixed